spec/spec_helper.rb

```diff
@@ -1,4 +1,10 @@
+require 'helpers'
+require 'paperclip/matchers'
+
 RSpec.configure do |config|
+  config.include Helpers
+  config.include Paperclip::Shoulda::Matchers
+
```

Comment:
This stuff should be moved to the support folder which will auto-included by rails_helper.rb - something along the lines of this:

``` ruby
# spec/support/paperclip.rb

module PaperclipHelpers
  def commons_image_fixture_path(filename)
    Rails.root.join('spec', 'fixtures', 'images', 'debate_outcome', filename)
  end

  def commons_default_image_url
    ActionController::Base.helpers.image_url('graphics/graphic_house-of-commons.jpg')
  end

  def commons_image_file
    commons_image_fixture_path 'commons_image-2x.jpg'
  end

  def commons_image_file_too_small
    commons_image_fixture_path 'commons_image-too-small.jpg'
  end

  def commons_image_file_wrong_ratio
    commons_image_fixture_path 'commons_image-wrong-ratio.jpg'
  end
end

RSpec.configure do |config|
  config.include PaperclipHelpers
  config.include Paperclip::Shoulda::Matchers
end
```